### ISSUE TITLE
Go Back command is not applied to the active editor when the file is open twice on different groups.

### ISSUE THREAD

**Issue by alfonso-soto** [microsoft/vscode#107291]
- VSCode Version: 1.49.1
- OS Version: Linux x64 4.15.0-72generic

The Go Back command is applied to the last cursor position on the file and not on the last cursor position of the file view.
I think this behavior is annoying when you have opened the same file twice to look at a specific line and edit some others on a different view, under this situation if at some point you click or copy over the first view and come back two the second one and use Go Back, you would be sent to the last cursor position of the first view.

In other words, the historical position of the cursor is not independent from the file view.

Steps to Reproduce:
1. Open the same file twice on the same vsCode window.
2. Move the cursor to multiples positions on the second views of the file.
3. Use the `Go back` command over the first view.
4. The first view would jump to the previous location of the second view.

Does this issue occur when all extensions are disabled?: Yes
